### PR TITLE
Edits about users, goal, and benefits of Quire

### PR DESCRIPTION
**Pull request by ggarcia0596**

### CODE REVIEW

**Comment by geealbers** [comment]
Welcome to your first pull request @ggarcia0596! Technically everything looks okay here, but this text was actually meant for [the About page](https://github.com/gettypubs/quire/blob/master/content/guide/about.md), rather than the README. If you can make that change on your `GG_About_Section_Edits_Describing_Quire` branch and push them back to GitHub, they'll be added to this pull request and I'll re-review. Let me know if you have any questions. Thanks!

File: README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 Quire is a modern, multi-format publishing framework designed to create books as authoritative and enduring as print, and as vibrant and feature-rich as the web, from a single set of plain text files. All without ever signing-up for an account, paying a fee, or setting up and maintaining a complicated server.
 
+Quire is a tool for publishing professionals and private individuals with an interest in digital publishing and everyone in between. Quire’s versatility lends itself to a variety of projects and users of all technology levels.
```
Comment (geealbers) [inline README.md]: Re-reading this now, I think "users of all technology levels" isn't quite accurate. There are more technical hurdles to working with Quire (at least in it's current state) than the average user might be comfortable. Can you reword?

**Comment by ggarcia0596** [comment]
Whoops. Yeah I'll move those to the correct section sorry about that. And for the rewording would something like: **"Quire’s versatility lends itself to a variety of projects and a diverse range of users."** be more appropriate? It's a little more vague so that there's not a specific distinction of who may be able to use it/figure it out and who may not, since some of your average users may be able to get a hang out it quickly.

**Comment by geealbers** [comment]
That rewording sounds ok, thanks.

**Comment by geealbers** [comment]
Great. Merging the request!

**Pull request merged**